tests/integration/targets/mount/tasks/main.yml

```diff
@@ -294,3 +294,37 @@
     - /tmp/myfs.img
     - /tmp/myfs
     when: ansible_system in ('Linux')
+
+- name: Block to test boot option for Linux
+  block:
+  - name: Create empty file
+    command: dd if=/dev/zero of=/tmp/myfs.img bs=1048576 count=20
```

Comment:
please consider using `community.general.filesize` rather than `command`:
```yaml
- name: Create empty file
  community.general.filesize:
    path: /tmp/myfs.img
    size: 20M
```

Reply:
Really appreciate your advice. I'll change it from `dd` to `community.general.filesize`